dg_test.go

```diff
@@ -182,6 +182,22 @@ func TestDirectedGraph_OneAndDependencyConnectDependency(t *testing.T) {
 		})
 }
 
+func TestDirectedGraph_ResolvingSingleNode(t *testing.T) {
+	// This test makes sure that it's only dependent nodes that get marked as ready.
+	// Since there are no dependencies connected here, there should be no nodes marked as ready.
+	d := dgraph.New[string]()
+	resolvedNode, err := d.AddNode("resolved-node", "resolved-node")
+	unresolvableNode, err := d.AddNode("unresolvable-node", "unresolvable-node")
+	assert.NoError(t, err)
```

Comment:
The linter points out that you're missing an assertion after line 189.

Reply:
Fixed.